packages/core/src/scss/components/checkout/stripeLink/_stripeDefaultValues.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../../../../src/app/ui/base";
+
+// We don't use these styles for the floating labels in the iframe.
+// These styles are in case of fallback for stripeIframe.scss
+// -----------------------------------------------------------------------------
+// stylelint-disable-next-line selector-pseudo-class-no-unknown
+:export {
+    border-radius: 4px;
+    font-size: 16px;
+    padding: 6px 12px 3.3px 12px;
+    padding-left: 4.3px;
+    padding-right: 2.8px;
```

Comment:
Just wondering, how did you come up with those decimal padding values?
Why are they different from that in `_stripeIframe.scss`?

Reply:
I saw this line `typeof stripeIframe === 'object' ? stripeIframe : defaultIframeValues`.
Should `typeof stripeIframe === 'object'` always be `true` since you have the full control here, if I am not mistaken?

Reply:
@bc-peng, the reason we use something by default that only stripeLink could manipulate is to avoid the use of dynamic data (without sass variables)

The decimal padding values are the result that I get when I inspected the elements, I use a default value only in case there is a failure or an accidental change in the sass variables, the idea is that these styles will never be used, I just add them as an extra precaution, since the component usually(depending of the browser) looks very bad without  styles, especially if the `floating labels` is activated, if you feel that it can make any trouble we can delete it :D

And regarding the last question you are correct, an apology, it was from a previous approach, indeed the sentence does not make sense, in this case, it should be `||` since in case of not sending styles the default value will be used, I will fix it